package/scripts/ambari_helpers.py

```diff
@@ -16,15 +16,18 @@
 from resource_management import *
 import os
 
+
 def create_hdfs_dir(path, owner, perms):
     Execute("hadoop fs -mkdir -p %s" % (path), user='hdfs')
     Execute("hadoop fs -chown %s %s" % (owner, path), user='hdfs')
     Execute("hadoop fs -chmod %s %s" % (str(perms), path), user='hdfs')
```

Comment:
Well, perms could be an integer, which is why I am converting. Maybe it's not needed?